ejb3/src/main/java/org/jboss/as/ejb3/component/pool/StrictMaxPoolConfigService.java

```diff
@@ -85,6 +87,10 @@ public synchronized void setMaxPoolSize(int newMax) {
         }
     }
 
+    public synchronized int getDerivedSize() {
```

Comment:
minor quip, you would not need to store the derivedSize if the `StrictMaxPoolConfigService` was implementing `Service<StrictMaxPoolConfig>` instead of `Service<PoolConfig>` so that you could get the actual value from the service by calling its `getValue().getMaxPoolSize()` instead